### PR TITLE
Updated outdated path.py dependency name

### PR DESCRIPTION
**Pull request by jmwright**
OS: Ubuntu 20.04 64-bit, Python 3.8.6

The `path.py` dependency in setup.py is the old package name which has since been renamed to just `path`. This was causing an older version of the path module to be installed, which was causing Python to throw an error because it expected a `0o` prefix in `mkdir()` instead of just a leading `0`.

[Old Path Package](https://pypi.org/project/path.py/)
[New Path Package](https://pypi.org/project/path/)

Here's the full stack trace (Ubuntu 20.04) that led to this PR.

```
$ python3 -m bindgen -n 1 -v parse ocp.toml out.pkl && python -m bindgen -n 1 -v transform ocp.toml out.pkl out_f.pkl && python -m bindgen -n 1 -v generate ocp.toml out_f.pkl
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.8/runpy.py", line 144, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/usr/lib/python3.8/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/home/jwright/Downloads/pywrap/bindgen/__init__.py", line 11, in <module>
    from path import Path
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 914, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1342, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1316, in _get_spec
  File "<frozen importlib._bootstrap_external>", line 1297, in _legacy_get_spec
  File "<frozen importlib._bootstrap>", line 414, in spec_from_loader
  File "<frozen importlib._bootstrap_external>", line 649, in spec_from_file_location
  File "<frozen zipimport>", line 191, in get_filename
  File "<frozen zipimport>", line 713, in _get_module_code
  File "<frozen zipimport>", line 647, in _compile_source
  File "/home/jwright/venvs/ocp/lib/python3.8/site-packages/path-2.2-py3.8.egg/path.py", line 886
    def mkdir(self, mode=0777):
                            ^
SyntaxError: leading zeros in decimal integer literals are not permitted; use an 0o prefix for octal integers
```

### CODE REVIEW

**Review by adam-urbanczyk — APPROVED** [review]

**Comment by jmwright** [comment]
@adam-urbanczyk Shall I also pin the jinja2 version in setup.py before you merge this?

**Comment by adam-urbanczyk** [comment]
Yes, good idea.

**Comment by marcus7070** [comment]
I also couldn't rerun anything, but I have a single login and get no errors on login. :shrug: Close and reopen will have to do.

**Comment by jmwright** [comment]
Thanks @marcus7070 it looks like all checks pass now. @adam-urbanczyk I think this should be ready to merge unless you see something that needs changed.

**Comment by adam-urbanczyk** [comment]
Looks good, thanks @jmwright

**Pull request merged**
